rpc/src/v1/impls/light/trace.rs

```diff
@@ -31,7 +31,7 @@ pub struct TracesClient;
 impl Traces for TracesClient {
 	type Metadata = Metadata;
 
-	fn filter(&self, _filter: TraceFilter) -> Result<Option<Vec<LocalizedTrace>>, Error> {
+	fn filter(&self, _filter: TraceFilter, _offset: Option<TraceIndex>, _count: Option<usize>) -> Result<Option<Vec<LocalizedTrace>>, Error> {
```

Comment:
wouldn't offset being `usize` be easier to use (and to implement)?

With offset being `TraceIndex` you can't fetch traces in parallel and/or from multiple clients. I imagine this might be a common practice for exchanges.

// thread 0, 
ethereum_client_0.filter(filter_options, 0, 1000);

// thread 1,
ethereum_client_1.filter(filter_options, 1000, 2000);

// thread 2,
ethereum_client_2.filter(filter_options, 2000, 3000);